src/main/scala/com/monsanto/arch/cloudformation/model/resource/IAM.scala

```diff
@@ -83,9 +83,8 @@ object `AWS::IAM::ManagedPolicy` extends DefaultJsonProtocol {
   implicit val format: JsonFormat[`AWS::IAM::ManagedPolicy`] = jsonFormat8(`AWS::IAM::ManagedPolicy`.apply)
 }
 
-
-case class AWSManagedPolicy(name: String) {
-  def buildARN = s"arn:aws:iam::aws:policy/$name"
+case class AWSManagedPolicy(name: Token[String], accountId : Token[String] = "aws") {
```

Comment:
Just based on naming, I'd probably be inclined to say let's have a separate case class, called something like `ExternalManagedPolicy`, `ExistingManagedPolicy`, or whatever.  `AWSManagedPolicy` was intended to be for a "Managed Policy" that's managed by AWS.  It wouldn't be the end of the world to keep it as you've submitted it, but it'd be somewhat clearer to go the other way.